src/EnergyPlus/HVACManager.cc

```diff
@@ -72,6 +72,7 @@
 //#include <CoolTower.hh>
 #include <DataAirflowNetwork.hh>
 #include <DataAirLoop.hh>
+#include <DataBranchAirLoopPlant.hh>
 #include <DataContaminantBalance.hh>
 #include <DataConvergParams.hh>
 #include <DataEnvironment.hh>
```

Comment:
Don't ask me why, even though I just pulled develop and then switched to this branch for the first time, I still had to pull from this branch to get these changes. Now things are lining up.